src/pages/evm/staking/StakingPageHeader.vue

```diff
@@ -3,105 +3,173 @@ import { computed, onBeforeMount, ref } from 'vue';
 import { useI18n } from 'vue-i18n';
 import { BigNumber, ethers } from 'ethers';
 
-import { useUserStore, useBalancesStore, useChainStore } from 'src/antelope';
-import { prettyPrintCurrency } from 'src/antelope/stores/utils/currency-utils';
+import { useUserStore, useBalancesStore, useChainStore, useRexStore } from 'src/antelope';
+import { convertCurrency, prettyPrintCurrency } from 'src/antelope/stores/utils/currency-utils';
 import { NativeCurrencyAddress } from 'src/antelope/types';
 import EVMChainSettings from 'src/antelope/chains/EVMChainSettings';
 import { WEI_PRECISION, formatWei } from 'src/antelope/stores/utils';
 
 import ScrollableInfoCards from 'components/evm/ScrollableInfoCards.vue';
 
-
+const label = 'current';
 const { t: $t } = useI18n();
 const userStore = useUserStore();
 const balancesStore = useBalancesStore();
 const chainStore = useChainStore();
 const chainSettings = (chainStore.loggedChain.settings as EVMChainSettings);
-// data
-const systemTokenPrice = ref(0); // always equal to wrapped token price
-const loading = ref(true);
-
 
 // computed
 const fiatLocale = computed(() => userStore.fiatLocale);
 const fiatCurrency = computed(() => userStore.fiatCurrency);
-const systemTokenName = computed(() => chainSettings.getSystemToken().name);
-const systemTokenSymbol = computed(() => chainSettings.getSystemToken().symbol);
-const stakedTokenSymbol = computed(() => chainSettings.getStakedSystemToken().symbol);
+const systemToken = chainSettings.getSystemToken();
+const stakedToken = chainSettings.getStakedSystemToken();
+
+
+// First cell: Staked
+const stakedTokenBalanceBn = computed(() => balancesStore.getBalances(label).find(balance => balance.token.symbol === stakedToken.symbol)?.amount);
```

Comment:
here and line 40 have an issue I think, they will not update reactively because it is set based on a single function call rather than a getter, like `balancesStore.getBalances(label)`. maybe i am mistaken but it seems to me that this will cause `stakedTokenBalanceBn` and `systemTokenPrice` to become stale. maybe `balancesStore.getBalances(label)` should be assigned to a data variable, and that variable should be updated on a timeout

Reply:
good observation !!
I changed those `computed` for `ref` and added a watch over the balance to update those refs.
Thanks!